server.js

```diff
@@ -1,38 +1,100 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-import mongoose from 'mongoose'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+import express from "express";
+import bodyParser from "body-parser";
+import cors from "cors";
+import mongoose from "mongoose";
+
+import netflixData from "./data/netflix-titles.json";
+
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/netflix-titles";
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.Promise = Promise;
+
+const Director = mongoose.model("Director", {
+  _id: mongoose.Schema.Types.ObjectId,
+  name: String,
+  titles: [{ type: mongoose.Schema.Types.ObjectId, ref: "Title" }]
+});
+
+const Title = mongoose.model("Title", {
+  title: String,
+  director: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "Director"
+  },
+  cast: String,
+  country: String,
+  release_year: Number,
+  type: String
+});
+
+const seedDatabase = async () => {
+  await Director.deleteMany();
+  await Title.deleteMany();
+
+  const distinct = (value, index, self) => {
+    return self.indexOf(value) === index;
+  };
+
+  const uniqNetflixData = netflixData.filter(title => title.director !== "");
+
+  const directors = uniqNetflixData.map(title => title.director).filter(distinct);
+
+  directors.forEach(async director => {
+    const newDirector = new Director({
+      _id: mongoose.Types.ObjectId(),
+      name: director
+    });
+
+    const directorsTitles = netflixData.filter(
+      title => title.director !== "" && title.director === director
+    );
+
+    directorsTitles.forEach(async title => {
+      const newTitle = new Title({
+        title: title.title,
+        director: newDirector._id,
+        cast: title.cast,
+        country: title.country,
+        release_year: title.release_year,
+        type: title.type
+      });
```

Comment:
Is there duplicate data in the database where the `Title` information is the same but only the `director` field is different?